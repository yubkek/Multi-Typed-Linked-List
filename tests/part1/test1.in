NEW 3
1
c
32e2
NEW 4
a
b
character
324
VIEW 0
TYPE 1
VIEW ALL
REMOVE 1
VIEW ALL
