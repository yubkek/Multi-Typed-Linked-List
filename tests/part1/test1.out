List 0: 1 -> c -> 3200.00
List 1: a -> b -> character -> 324
1 -> c -> 3200.00
char -> char -> string -> int
Number of lists: 2
List 0
List 1
List 1 has been removed.

Number of lists: 1
List 0
Number of lists: 1
List 0